docs/downloads/index.md

```diff
@@ -1,6 +1,10 @@
 ---
 layout: default
 ---
+
+{% assign versionSplit = site.data.package.version | split: "." %}
+{% assign majorVersion = versionSplit[ 0 ] %}
+
 # Downloads
 
 The SDK is available as CommonJS, AMD, and global modules. CDN-hosted versions are available on the Shopify CDN, or you can download the package from NPM or Github.
```

Comment:
Should be good to go
https://github.com/Shopify/js-buy-sdk/blob/docs-cdn-url-update/docs/downloads/index.md#downloads